cam1_tf: [0.667691, -0.342270, 0.505848, 0.010304, 0.543505, -0.002211, 0.839340]
cam2_tf: [0.153252, -0.0757284, 0.384213, 0.909495, -0.0193283, -0.41493, -0.016694]
